schema/gkrule-source.yaml

```diff
@@ -0,0 +1,144 @@
+$schema: "http://json-schema.org/draft/2020-12/schema"
+title: "GA4GH-Genomic-Knowledge-Rule-Definitions"
+type: object
+namespaces:
+  vrs: "https://raw.githubusercontent.com/ga4gh/vrs/1.2.0/schema/vrs.json#/definitions/"
+
+$defs:
+  # VRSATILE Genomic Knowledge Rules are presented top-down. Everything rolls up to
+  # GenomicKnowledgeRule, which is a polymorphic abstraction of many kinds of genomic
+  # knowledge rules.
+
+  # =============================================================================
+  # Top-level Genomic Knowledge Rule concepts
+  # =============================================================================
+
+  GenomicKnowledgeRule:
+    description: >-
+      The abstract *Genomic Knowledge Rule* parent class. All attributes of this parent
+      class are inherited by Genomic Knowledge Rule descendent classes. Genomic Knowledge
+      Rules are a mechanism for specifying constraints on how observed variation matches
+      to genomic knowledge.
```

Comment:
change `to genomic knowledge` to `to rules based on aspects of genomic attributes and characteristics`.  
We are not really matching to `knowledge` per se (are we)?